libexec/dyld/aarch64/rtld_start.S

```diff
@@ -128,10 +126,8 @@ END(_rtld_bind_start)
  * Resolver function for TLS symbols resolved at load time
  */
 ENTRY(_rtld_tlsdesc_static)
-	.cfi_startproc
```

Comment:
Does this not work on arm64, or not needed? Just curious for my future work on it.

Reply:
Hmm, OK. I'm probably going to scrap this hybrid dyld anyway and try to get Apple's working as part of the move towards Mach-O, so no big deal.